### PR TITLE
Feature/user car current color fix

### PR DESCRIPTION
**Pull request by mvrsdev**
Fixed de userCar currentColor to 'Grey' instead of pulling from the array, witch was causing the script to return with undesireable behaviour.

### CODE REVIEW

File: scripts/car.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ const userCar = {
     'Grey', 
     'Blue'
   ],
-  currentColor: availableColors[4],
-  engine: '2.0',
-  transmission: 'Powershift 6 marchas + ré',
+  currentColor: 'Grey',
+  // engine: 1234,
```
Comment (thi-days) [inline scripts/car.js:16]: Change its commented code to an empty object:

`engine: {}`

**Review by thi-days — CHANGES_REQUESTED** [review]

File: scripts/car.js (hunk 1 above)
Comment (mvrsdev) [reply scripts/car.js]: Done!

File: scripts/car.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ const userCar = {
...
   ],
-  currentColor: availableColors[4],
-  // engine: 1234,
-  // transmission: 1234,
-  // imagePath: 1234,
-  // dimensions: 1234,
-  // safety: 1234,
-  // performance: 1234,
+  currentColor: 'Grey',
+  engine: ,
```
Comment (thi-days) [inline scripts/car.js:16]: É necessário colocar um empty object { }. Uma linha vazia dá erro.

**Review by thi-days — CHANGES_REQUESTED** [review]

File: scripts/car.js (hunk 2 above)
Comment (mvrsdev) [reply scripts/car.js]: Now it's really fixed. My bad.

**Review by thi-days — APPROVED** [review]
It's okay now!

**Pull request merged**